src/views/HelpAndKnowledge/Articles/ModelPlanOverview/index.tsx

```diff
@@ -44,12 +45,12 @@ export const ModelPlanOverviewContent = ({
           </Trans>
         </p>
       </SummaryBox>
-      <PageHeading className="margin-top-7 margin-bottom-1">
+      <PageHeading className="margin-top-7 margin-bottom-1 font-body-xl">
```

Comment:
This is still rendered as an `<h1>` element.  This should be replaced to that element isn't used.  Should be an `<h2>` element